cpp/velox/memory/VeloxMemoryManager.h

```diff
@@ -44,13 +44,22 @@ class VeloxMemoryManager final : public MemoryManager {
   }
 
  private:
+  facebook::velox::memory::IMemoryManager::Options getOptions(
+      std::shared_ptr<MemoryAllocator> allocator,
+      std::shared_ptr<AllocationListener> listener) const;
+
   std::string name_;
 
+#ifdef GLUTEN_ENABLE_HBM
+  std::unique_ptr<VeloxMemoryAllocator> wrappedAlloc_;
+#endif
```

Comment:
Member order: This better be placed after `glutenAlloc_` ?

Reply:
seems to be no relation between these two data members